### PR TITLE
CLDC-825: Support suffixes and prefixes for numeric fields

### PR DESCRIPTION
**Pull request by baarkerlounger**
- [x] Income frequency is derived rather than asked
- [x] Prefix and suffix can be added to JSON and is displayed
- [x] Prefix/suffix/header updates based on previous answer
- [x] Readme is updated

Weekly:
image
image

Monthly:
image
image

Yearly:
image
image

### CODE REVIEW

File: app/models/case_log.rb [hunk 1]
```diff
@@ -215,7 +215,16 @@ def set_derived_fields
...
+    case net_income_known
+    when "Yes – the household has a weekly income"
+      self.incfreq = "Weekly"
+    when "Yes – the household has a monthly income"
+      self.incfreq = "Monthly"
+    when "Yes – the household has a yearly income"
+      self.incfreq = "Yearly"
+    when "Tenant prefers not to say"
+      self.incref = 1
+    end
```
Comment (kosiakkatrina) [inline app/models/case_log.rb:227]: I don't know if it'd look better or not, but what if we have a mapping instead?
```
INCREF_MAPPINGS =  {"Yes – the household has a weekly income" : "Weekly",  "Yes – the household has a monthly income": "Monthly" ...}

self.incfrec = INCREF_MAPPINGS[net_income_known]
```

Comment (baarkerlounger) [reply app/models/case_log.rb:227]: Happy to change but note there's two different fields being set there depending on the case (`incref` and `incfreq` so might not get rid of that much)

Comment (kosiakkatrina) [reply app/models/case_log.rb:227]: Oh, yeah, that's fair. I don't have a strong preference

**Review by kosiakkatrina — APPROVED** [review]
Looks good! Just one thing that is not part of this PR but should probably be fixed. We're saving the amount as integers, so £34.15 would round to £34

**Comment by baarkerlounger** [comment]
@kosiakkatrina that's expected isn't it? We only want to the nearest pound (although that's no longer mentioned in the question text, cc. @Dan13L£)

**Comment by kosiakkatrina** [comment]
> @kosiakkatrina that's expected isn't it? We only want to the nearest pound (although that's no longer mentioned in the question text, cc. @dan13l£)

Oh, I'm not sure anymore..

**Pull request merged**